### ISSUE TITLE
[BUG]: ImportError on python examples

### ISSUE THREAD

**Issue by chamcca** [aws-cloudformation/aws-cloudformation-samples#85]
Python example fails to execute and unit tests fail with the error:

 ImportError: cannot import name 'verify_interface' from 'cryptography.utils'

**Comment by mrinaudo-aws** [comment]
Closing the issue and the PR accordingly, per above.

**Comment by mrinaudo-aws** [comment]
No worries at all!  Thank you for jumping in.  The updated lib (and bumped up version number) should cause the `cloudformation-cli-python-lib>=2.1.3` current line in the `requirements.txt` on this sample to consume from the latest, to-be-released version of `cloudformation-cli-python-lib`, whose `aws-encryption-sdk` version I see it to be set to `3.1.1` as per [here](https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/blob/master/src/setup.py#L18).  With this holding true, the long-term need for forcing that dependency on a target extension (for example, this sample resource type) would not be needed, as it should be inherited by the latest, to-be-released version of that library.

**Comment by mrinaudo-aws** [comment]
Hi @chamcca - the Python plugin and lib are now updated.  I used `pip` to upgrade them on my machine, where I now have `aws-encryption-sdk-3.1.1`, `cloudformation-cli-python-lib-2.1.16`, and `cloudformation-cli-python-plugin-2.1.8`.  With no changes to my `requirements.txt` file for the sample resource type in question (meaning, no need to add the temporary line mentioned earlier), I was able to rebuild (`cfn generate && cfn submit --dry-run`), and then to successfully run contract tests (`13 passed, 2 skipped, 9 deselected, 1 warning in 164.10s (0:02:44)`, and `13 passed, 2 skipped, 9 deselected, 1 warning in 151.41s (0:02:31)`).  You as well should then be able to expunge the aforementioned line in your `requirements.txt` file, as it is now part of the "inherited" dependency; what I have now is only:

```
% cat requirements.txt 
cloudformation-cli-python-lib>=2.1.3
```

Thank you!
